commons/src/main/java/io/hyscale/commons/models/YAMLManifest.java

```diff
@@ -19,14 +19,14 @@
 
 public class YAMLManifest implements Manifest {
 
-	private File yamlManifest;
+	private File manifestFile;
```

Comment:
It's already evident that it is a file with Data Type , so just manifest would suffice here.